MTM-46453 Added requirements for all major cockpit features. (#532)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/managing-assets.md

Groups show the following tabs:

- **Subassets** - Shows group details and all subassets of a group, see also [Device management > Viewing devices](/users-guide/device-management/#viewing-devices).
- **Smart rules** - Shows smart rules specified for the group, see also [Smart rules](#smart-rules).
- **Data explorer** - Shows all data points of the children. For details refer to [Visualizing data using the data explorer](#data-explorer).

{{< c8y-admon-req >}}
ROLES & PERMISSIONS in groups context:

- To view all groups: READ permission for permission type "Inventory"
- To add new groups: CREATE permission for permission type "Inventory"
- To delete any group: ADMIN permission for permission type "Inventory"
- To rename a group or change group description: ADMIN permission for permission type "Inventory"
- To view specific groups: READ permissions for "Inventory" in the inventory roles
- To manage or delete specific groups: READ and CHANGE permissions for "Inventory" in the inventory roles 

Note that global inventory permissions override inventory role permissions.
{{< /c8y-admon-req >}}

Devices show the following tabs:

> - **Info** - Shows smart rules specified for the device, see also [Smart rules](#smart-rules).

Author: renamed

Groups show the following tabs:

- **Subassets** - Shows group details and all subassets of a group, see also [Device management > Viewing devices](/users-guide/device-management/#viewing-devices).
- **Smart rules** - Shows smart rules specified for the group, see also [Smart rules](#smart-rules).
- **Data explorer** - Shows all data points of the children. For details refer to [Visualizing data using the data explorer](#data-explorer).

{{< c8y-admon-req >}}

> ROLES & PERMISSIONS IN GROUPS CONTEXT:

Reviewer: ```suggestion
ROLES & PERMISSIONS in groups context:
```

> - To view all groups: READ or CREATE permission for permission type "Inventory"

Reviewer 2: Create permissions to view all groups?

Author: fixed

- To add new groups: CREATE permission for permission type "Inventory"
- To delete any group: ADMIN permission for permission type "Inventory"
- To rename a group or change group description: ADMIN permission for permission type "Inventory"

> - To view specific groups: READ permissions for permission type "Inventory" in the Inventory role permissions.

Reviewer: ```suggestion
- To view specific groups: READ permissions for permission type "Inventory" in the Inventory role permissions
```

Author: fixed

> - To manage or delete specific groups: READ and CHANGE permissions for permission type "Inventory" in the Inventory role permissions

Reviewer: ```suggestion
- To manage or delete specific groups: READ and CHANGE permissions for "Inventory" in the inventory roles 
```

> - To manage or delete specific groups: READ and CHANGE permissions for permission type "Inventory" in the Inventory role permissions.

Reviewer: ```suggestion
- To manage or delete specific groups: READ and CHANGE permissions for permission type "Inventory" in the Inventory role permissions
```

{{< /c8y-admon-req >}}

{{< c8y-admon-info >}}

> Global inventory permissions will override inventory role permissions.

Reviewer: I would move this into the permission box

{{< /c8y-admon-info >}}

Devices show the following tabs:

- **Info** - Shows smart rules specified for the device, see also [Smart rules](#smart-rules).
- **Alarms** - Displays alarms for the device, see also [Device management > Working with alarms](/users-guide/device-management/#alarm-monitoring).
- **Data explorer** - Shows all data points of the children. For details refer to [Visualizing data using the data explorer](#data-explorer).
- **Location** - Shows the current location of a device (only available with `c8y_Position`).

{{< c8y-admon-req >}}

> ROLES & PERMISSIONS IN DEVICES CONTEXT:

Reviewer: ```suggestion
ROLES & PERMISSIONS in devices context:
```

> - To view all devices within a group: READ or CREATE permission for permission type "Inventory"

Reviewer 2: CREATE shouldn't allow to view all devices..

Author: fxied

> - To assign or unassign devices within a group: CREATE permission for permission type "Inventory"

Reviewer 2: Sure this is not ADMIN permission?

- To delete any device within a group: ADMIN permission for permission type "Inventory"

>   {{< /c8y-admon-req >}}

Reviewer 3: ```suggestion
{{< /c8y-admon-req >}}
```
